packages/xo-server-sdn-controller/src/index.js

```diff
@@ -701,6 +992,133 @@ class SDNController extends EventEmitter {
     })
   }
 
+  // ---------------------------------------------------------------------------
+
+  async _clearCenterPoolFromNetwork(
+    centerPoolNetwork,
+    poolNetwork,
+    crossPoolNetworkUuid
+  ) {
+    const centerClient = find(
+      this._ovsdbClients,
+      client => client.host.$ref === centerPoolNetwork.starCenter
+    )
+    if (centerClient === undefined) {
+      return
```

Comment:
What does it mean, is it supposed to happen?

Reply:
Yes when no hosts from the pool are online, there can't be a center.
It comes back when one host is back online.